src/main/java/mil/dds/anet/beans/search/AttachmentSearchQuery.java

```diff
@@ -0,0 +1,14 @@
+package mil.dds.anet.beans.search;
+
+public class AttachmentSearchQuery extends AbstractSearchQuery<AttachmentSearchSortBy> {
+
```

Comment:
`authorUuid` should be added as a field here, and used in `AbstractAttachmentSearcher::buildQuery`.